tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraFilterStateChangeTest.scala

```diff
@@ -0,0 +1,180 @@
+package com.linagora.tmail.james.common
+
+import com.google.inject.AbstractModule
+import com.google.inject.multibindings.Multibinder
+import com.linagora.tmail.james.jmap.model.FilterTypeName
+import io.restassured.RestAssured.requestSpecification
+import net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson
+import org.apache.james.GuiceJamesServer
+import org.apache.james.jmap.api.model.AccountId
+import org.apache.james.jmap.change.{StateChangeEvent, TypeName}
+import org.apache.james.jmap.draft.JmapGuiceProbe
+import org.apache.james.jmap.http.UserCredential
+import org.apache.james.jmap.rfc8621.contract.Fixture.{ACCEPT_RFC8621_VERSION_HEADER, BOB, BOB_PASSWORD, DOMAIN, authScheme, baseRequestSpecBuilder}
+import org.apache.james.jmap.rfc8621.contract.{CustomMethodContract, JmapEventBusProbe}
+import org.apache.james.utils.{DataProbeImpl, GuiceProbe}
+import org.junit.jupiter.api.{BeforeEach, Test}
+import sttp.capabilities.WebSockets
+import sttp.client3.monad.IdMonad
+import sttp.client3.okhttp.OkHttpSyncBackend
+import sttp.client3.{Identity, RequestT, SttpBackend, asWebSocket, basicRequest}
+import sttp.model.Uri
+import sttp.monad.MonadError
+import sttp.monad.syntax.MonadErrorOps
+import sttp.ws.WebSocketFrame
+import sttp.ws.WebSocketFrame.Text
+
+import java.net.URI
+
+class FilterTypeNameModule extends AbstractModule {
```

Comment:
Arf again Quan these bindings are not specific to tests, you need to move it to the top level maven module declaration!

Overwize we will not have these modules in production!